class-wc-facebookcommerce.php

```diff
@@ -133,6 +135,9 @@ public function init() {
 			add_filter( 'fb_product_set_row_actions', array( $this, 'product_set_links' ) );
 			add_filter( 'manage_edit-fb_product_set_columns', array( $this, 'manage_fb_product_set_columns' ) );
 
+			// Hook the setup task.
+			add_action( 'init', array( $this, 'add_setup_task' ), 20 );
```

Comment:
I suggest we run the setup task on `admin_init` rather

```suggestion
			add_action( 'admin_init', array( $this, 'add_setup_task' ), 20 );
```